com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -49,51 +39,22 @@ internal struct AnimationTriggerMessage : INetworkSerializable
         }
 
         [SerializeField] private Animator m_Animator;
-        [SerializeField] private uint m_ParameterSendBits;
-        [SerializeField] private float m_SendRate = 0.1f;
 
         public Animator Animator
         {
             get { return m_Animator; }
             set
             {
                 m_Animator = value;
-                ResetParameterOptions();
-            }
-        }
-
-        /*
-         * AutoSend is the ability to select which parameters linked to this animator
-         * get replicated on a regular basis regardless of a state change. The thinking
-         * behind this is that many of the parameters people use are usually booleans
-         * which result in a state change and thus would cause a full sync of state.
-         * Thus if you really care about a parameter syncing then you need to be explicit
```

Comment:
Unfortunately I can't ask Andrew what he meant by "if you really care", but apart from adding redundancy to cover unreliable delivery (which we don't use for this), the only scenario I can think of here is if you could go from state A to state B and back to state A all between 2 sequential FixedUpdate calls (and also not using our SetTrigger), in which case we wouldn't pick it up and the connected players would stay on 'A' missing the ultra-short 'B' transition.